code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -418,7 +419,7 @@
 /obj/item/robot_module/peacekeeper/do_transform_animation()
 	..()
 	to_chat(loc, "<span class='userdanger'>Under ASIMOV, you are an enforcer of the PEACE and preventer of HUMAN HARM. \
-	You are not a security module and you are expected to follow orders and prevent harm above all else. Space law means nothing to you.</span>")
+	You are not a controller module and you are expected to follow orders and prevent harm above all else. Space law means nothing to you.</span>")
```

Comment:
wrong place, this is for peacekeeper borgs

Reply:
changed for the controller transform text also, this is intentional